crates/typst-syntax/src/lexer.rs

```diff
@@ -107,6 +111,15 @@ impl Lexer<'_> {
         }
     }
 
+    pub fn next_raw(&mut self) -> SyntaxKind {
```

Comment:
Is this a separate function in order to not slow down `next()` with a check for `self.offsets.is_empty()` every time?